api/v1alpha1/resourceflavor_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ResourceFlavorSpec defines the desired state of ResourceFlavor
+type ResourceFlavorSpec struct {
+	// resource is the resource name of the flavor, e.g., "cpu".
+	Resource corev1.ResourceName `json:"resource,omitempty"`
```

Comment:
What would be the usefulness of this field?

I could imagine cpu and memory pointing to the same flavor.

Reply:
yeah, I agree. Removed